test/framework/src/test/java/org/elasticsearch/test/VersionUtilsTests.java

```diff
@@ -147,24 +149,22 @@ public void testResolveReleasedVersionsForStableBtranchBehindStableBranch() {
     }
 
     static class TestUnstableBranch {
-        public static final Version V_5_3_0 = Version.fromString("5.3.0");
-        public static final Version V_5_3_1 = Version.fromString("5.3.1");
-        public static final Version V_5_3_2 = Version.fromString("5.3.2");
-        public static final Version V_5_4_0 = Version.fromString("5.4.0");
         public static final Version V_6_0_0_alpha1 = Version.fromString("6.0.0-alpha1");
         public static final Version V_6_0_0_alpha2 = Version.fromString("6.0.0-alpha2");
         public static final Version V_6_0_0_beta1 = Version.fromString("6.0.0-beta1");
-        public static final Version CURRENT = V_6_0_0_beta1;
```

Comment:
The intent of these tests was to not have to change them when we upgraded unless we found a new kind of layout. So maybe we don't have to change this?